### ISSUE TITLE
Add optional `client-class-name` to API

### ISSUE THREAD

**Issue by zachkirsch** [fern-api/fern#878]
This will be used as the name of exported client class in the SDKs (mod casings)

**Comment by zachkirsch** [comment]
default to org name

**Comment by dsinghvi** [comment]
As internal users of Fern, we'll have to remember to override b/c itll default to org name. I don't really see how we get around that with the default

**Comment by zachkirsch** [comment]
+1, unless we have a concept of internal/external which changes the default per-generator

**Comment by dsinghvi** [comment]
+1

**Comment by estahn** [comment]
@dsinghvi Any update on this? We started to use fern to generate APIs for external integrations and these should really have different names, instead of the org name, e.g. `FooApi`, `BarApi`, `MoreApi`, ...

**Comment by dsinghvi** [comment]
@estahn yeah this is a good time, for now you can use configs in each of the generator to get around this. `namespaceExport` in the ts sdk, `client_class_name` in the python sdk.

**Comment by dannysheridan** [comment]
Today a user's github username didn't make sense for the organization name but is auto-populated by Fern. 

Two possible solutions:
1. prompt the user for an organization name
2. use the title property from an openapi spec
